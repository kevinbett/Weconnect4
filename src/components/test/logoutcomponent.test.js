import Logout from '../LogoutComponent'
import { shallow } from "enzyme"

jest.mock('../../apiServices/api');

describe('Logout component',() => { 
    it ('Logout componet renders', (done) => {
        const wrapper =  shallow(<Logout/>);
        
        setTimeout(() => {
            wrapper.update();
            expect(wrapper).toMatchSnapshot();
            done();
        });
    });
});