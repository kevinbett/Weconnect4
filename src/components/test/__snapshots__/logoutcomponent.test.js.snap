// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout component Logout componet renders 1`] = `
<div>
  <div
    class="loader"
    id="loader-2"
  />
</div>
`;
